content/en/docs/collector/distributions.md

```diff
@@ -12,6 +12,8 @@ the `manifest.yaml` of each distribution.
 [distributions]:
   https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions
 
+{{% ecosystem/distributions-table filter="oss-collector" %}}
```

Comment:
Might `otel` be preferable to `oss`. Or maybe, `first-party`?

```suggestion
{{% ecosystem/distributions-table filter="otel-collector" %}}
```

Reply:
Maybe a small description of that the table contains would be useful, just before the table. Otherwise, at first glance, it makes me wonder why I wouldn't just visit the "pre-built distributions" link given in the first paragraph.